agriMushroom/ngsild-payloads/agriParcel.json

```diff
@@ -11,11 +11,11 @@
             ]
         }
     },
-    "cropStatus": {
+    "https://uri.fiware.org/ns/data-models#cropStatus": {
         "type": "Property",
         "value": "seeded"
     },
-    "lastPlantedAt": {
+    "https://uri.fiware.org/ns/data-models#lastPlantedAt": {

```

Comment:
same as the previous comment

Reply:
That's correct, I'm seeing this on a retrieve entity operation (whereas there is a LINK header to the context or not). We discussed this some days ago and we established it was a problem in the smart data models if I did not misunderstood, the reason why I wanted to highlight here by example.

Should I remove the expanded part instead ?